### PR TITLE
Remove proxy port from connection tracking table

### PR DESCRIPTION
**Pull request by tgraf**
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/3601)

### CODE REVIEW

**Comment by tgraf** [comment]
test-me-please

File: bpf/lib/common.h [hunk 1]
```diff
@@ -422,7 +422,6 @@ struct ct_state {
 	__u16 loopback:1,
 	      reserved:15;
 	__be16 orig_dport;
-	__be16 proxy_port;
```
Comment (joestringer) [inline bpf/lib/common.h:425]: Should this be replaced with padding?

Comment (tgraf) [reply bpf/lib/common.h:425]: It's not stored in any maps so I did not add padding. Any particular reason for the padding?

Comment (joestringer) [reply bpf/lib/common.h:425]: Nope, makes sense.

**Comment by eloycoto** [comment]
Backport https://github.com/cilium/cilium/pull/3638

**Pull request merged**